### PR TITLE
Master

### PR DESCRIPTION
**Pull request by sbonnick**
made changes as discussed in previous pull request: https://github.com/bear/parsedatetime/pull/127

### CODE REVIEW

**Comment by philiptzou** [comment]
Does that work in your PyCharm?

**Comment by sbonnick** [comment]
yep. I can confirm this fixes the issue.

File: parsedatetime/__init__.py [hunk 1]
```diff
@@ -271,6 +271,8 @@ def __init__(self, constants=None, version=VERSION_FLAG_STYLE):
         @param version:   Default style version of current Calendar instance.
                           Valid value can be 1 (L{VERSION_FLAG_STYLE}) or
                           2 (L{VERSION_CONTEXT_STYLE}). See L{parse()}.
+						  
+		@rtype:  Calendar
```
Comment (philiptzou) [inline parsedatetime/__init__.py]: I think this is 2 tabs but not spaces. Please fix it.

**Comment by bear** [comment]
for the record... anyone is allowed to tell me nicely "hey, umm, bear... that's a silly change ;)"

thanks for the issue @sbonnick and for the curation @philiptzou

**Pull request merged**